### ISSUE TITLE
Project configuration file not loaded

### ISSUE THREAD

**Issue by facundocapua** [lavary/crunz#108]
First of all, I want to congratulate you for the great module you've developed. The code looks clean and the documentation is great, great work!

I have a question regarding using my custom `crunz.yml`.

Let me summarise the steps I followed:
* I have installed the module via composer
```bash
composer require lavary/crunz
```

* Then I have create a new Tasks file under some project folder:

* Testing the executing like this:
```bash
vendor/bin/crunz schedule:run custom/folder/tasks
```
The result I have is this:
```bash
 [Deprecation] Timezone is not configured and this is deprecated from 1.7 and will result in exception in 2.0 version.
               Add `timezone` key to your YAML config file. File
```

So then I've done this:
```bash
vendor/bin/crunz publish:config
```

And as expected, a `crunz.yml` file was created in the root level directory.
I have changed the `source` folder in the new file, but when I execute the command, this is the result:
```bash
vendor/bin/crunz schedule:run
No task found! Please check your source path.
```

What am I doing wrong?

I've done some debugging, and it seems the problem is here:
https://github.com/lavary/crunz/blob/81ebcd400319d61a7079f611ad1c286ec2e483ab/src/Configuration/Configuration.php#L70

I have done a test on my installation, and changing that to:
```php
$config_file = getbase() . '/crunz.yml';
```

Seems to fix the problem, but I'm not sure it is the proper fix.

Let me know if that is the fix and I can submit a Pull Request.

Thanks a lot!

**Comment by monitaure** [comment]
I was wondering why my tasks did not launch, @facundocapua thanks for your issue.

On my projects `CRUNZ_ROOT` is setted as `/project/path/vendor/lavary/crunz` and `getbase` return `/project/path/vendor/lavary/../..`.

The `getbase` function's documentation says _Return the project's root directory._ so @facundocapua I think your fix `$config_file = getbase() . '/crunz.yml';` would be ok in https://github.com/lavary/crunz/blob/81ebcd400319d61a7079f611ad1c286ec2e483ab/src/Configuration/Configuration.php#L70
At least I will use it right now for my projects, thanks.

Thanks to the crunz contributors too.

**Comment by PabloKowalczyk** [comment]
Hello guys.
You are 100% right about bug. I think that first path to check should be current working directory, then `getbase() . '/crunz.yml';` and fallback to `CRUNZ_ROOT . '/crunz.yml'`.
I will work on fix and try to do release as fast as possible.
In the future i will add option to specify config file by `--config`.
Thanks for report guys.

**Comment by PabloKowalczyk** [comment]
Fixed and released as `v1.7.1`, thanks @facundocapua and @monitaure!

**Comment by facundocapua** [comment]
Thank you!! @PabloKowalczyk
